.github/workflows/ci.yml

```diff
@@ -32,3 +33,7 @@ jobs:
         run: |
           bower install
           npm run-script test --if-present
+
+
+      - name: Check formatting
+        run: purs-tidy check src test
```

Comment:
```suggestion
        run: purs-tidy check src test

```

Reply:
In the script, looks like I need to drop blank lines before adding this one in case there already is a blank line there...

Reply:
It's the opposite, no? This is missing its trailing blank line.

Reply:
Oh, yeah, I fixed that, too. But I also noticed the two extra blank lines above the `- name: Check formatting` line.

Reply:
Well, now it's fixed.